data/migrations/20210125195605_favorites.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function (knex) {
+  return knex.schema.createTable("favorites", tbl => {
+    tbl.integer('okta_email')
```

Comment:
Is there user value to calling this okta email? Are we capturing other emails? perhpas just call it email.